Documentation/seccomp-guide.md

```diff
@@ -0,0 +1,377 @@
+ * `@rkt/default-whitelist` represents a broad-scope filter than can be used for generic whitelisting
+
+For compatibility reasons, two groups are provided mirroring [default Docker
+profiles](https://github.com/docker/docker/blob/v1.12.0/docs/security/seccomp.md):
+
+ * `@docker/default-blacklist`
+ * `@docker/default-whitelist`
+
+When using stage1 images with systemd >= v231, some
+[predefined groups](https://www.freedesktop.org/software/systemd/man/systemd.exec.html#SystemCallFilter=)
+are also available:
+
+ * `@systemd/clock` for syscalls manipulating the system clock
+ * `@systemd/default-whitelist` for a generic set of typically whitelisted syscalls
+ * `@systemd/mount` for filesystem mounting and unmounting
+ * `@systemd/network-io` for socket I/O operationgs
+ * `@systemd/obsolete` for unusual, obsolete or unimplemented syscalls
+ * `@systemd/privileged` for syscalls which need super-user syscalls
+ * `@systemd/process` for syscalls acting on process control, execution and namespacing
+ * `@systemd/raw-io` for raw I/O port access
+
+When no seccomp filtering is specified, by default rkt whitelists all the generic
+syscalls typically needed by applications for common operations. This is
+the same set defined by `@rkt/default-whitelist`.
+
+The default set is tailored to stop applications from performing a large
+variety of privileged actions, while not impacting their normal behavior.
+Operations which are typically not needed in containers and which may
+impact host state, eg. invoking `umount(2)`, are denied in this way.
+
+However, this default set is mostly meant as a safety precaution against erratic
+and misbehaving applications, and will not suffice against tailored attacks.
+As such, it is recommended to fine-tune seccomp filtering using one of the
+customizable isolators available in rkt.
+
+## Seccomp Isolators
+
+When running Linux containers, rkt provides two mutually exclusive isolators
+to define a seccomp filter for an application:
+
+* `os/linux/seccomp-retain-set`
+* `os/linux/seccomp-remove-set`
+
+Those isolators cover different use-cases and employ different techniques to
+achieve the same goal of limiting available syscalls. As such, they cannot
+be used together at the same time, and recommended usage varies on a
+case-by-case basis.
+
+### Operation mode
+
+Seccomp isolators work by defining a set of syscalls than can be either blocked
+("remove-set") or allowed ("retain-set"). Once an application tries to invoke
+a blocked syscall, the kernel will deny this operation and the application will
+be notified about the failure.
+
+By default, invoking blocked syscalls will result in the application being
+immediately terminated with a `SIGSYS` signal. This behavior can be tweaked by
+returning a specific error code ("errno") to the application instead of
+terminating it.
+
+For both isolators, this can be customized by specifying an additional `errno`
```

Comment:
Right! I've slightly adapted the existing examples to also show `errno` in https://github.com/coreos/rkt/pull/3033/commits/1bdf84d712fc9c9150fda4ef95653faad28f4ae4 without adding too much detour, is it clear enough?